miden-lib/src/tests/test_prologue.rs

```diff
@@ -519,3 +502,10 @@ fn test_get_blk_timestamp() {
 
     assert_eq!(process.stack.get(0), tx_inputs.block_header().timestamp());
 }
+
+// HELPER FUNCTIONS
+// ================================================================================================
+
+fn read_root_mem_value(process: &Process<MockHost>, addr: u32) -> Word {
+    process.get_mem_value(ContextId::root(), addr).unwrap()
```

Comment:
Should we return an error instead of `unwrap()`?

Reply:
This is testing code - so, `unwrap()` is fine.